test/selector-validation.js

```diff
@@ -235,6 +235,47 @@ describe('selector validation', function() {
     }
   );
 
+  describe(
+    'with different `initial` ("^\\.prefix$") and ' +
+    '`combined` ("^\\.prefix-" + cmpt + "(?:-[a-z]+)?$") selector patterns, ' +
+    'where the initial selector does not contain the component name',
+    function() {
+      describe('as regular expressions', function() {
+        runTests({
+          initial: function(cmpt) {
+            return new RegExp('^\\.prefix$');
+          },
+          combined: function(cmpt) {
+            return new RegExp('^\\.prefix-' + cmpt + '(?:-[a-z]+)?$');
+          },
+        });
+      });
+      describe('as strings', function() {
+        runTests({
+          initial: '^\\.prefix$',
+          combined: '^\\.prefix-{componentName}(?:-[a-z]+)?$',
+        });
+      });
+
+      function runTests(componentSelectors) {
+        var patternB = { componentSelectors: componentSelectors };
```

Comment:
No, that variable name is just the result of copy & paste from one of the existing tests :-).